base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
oh maybe the syntax used in the doc index does matter then? kinda guessing, someone who's more familiar with Documenter should know better